dxfeed/wrappers/DXFeed.py

```diff
@@ -0,0 +1,54 @@
+from dxfeed.core.DXFeedPy import *
+from dxfeed.wrappers.DXFeedSubscription import DXFeedSubscription
+
+
+class DXFeed(object):
+    """
+    Class for connection management. After successful creation Instance will be connected to server
+    with provided credentials
+
+    Attributes
+    ----------
+    connection_address: str
+        One of possible connection addresses:
+        
+        - the single address: `host:port` or just `host`
+        - address with credentials: `host:port[username=xxx,password=yyy]`
+        - multiple addresses: `(host1:port1)(host2)(host3:port3[username=xxx,password=yyy])`
+        
+        Default: demo.dxfeed.com:7300
+    """
+    def __init__(self, connection_address: str = 'demo.dxfeed.com:7300'):
+        self.__con_address = connection_address
+        self.__connection = ConnectionClass()
+        self.connect()
+
+    @property
+    def connection_status(self):
+        return dxf_get_current_connection_status(self.__connection, return_str=True)
+
+    @property
+    def address(self):
+        return dxf_get_current_connected_address(self.__connection)
+
+    def connect(self):
```

Comment:
It would be better to add additional argument to indicate necessity of reconnection in case of existing connection.